src/computation/logic/district-merging.ts

```diff
@@ -0,0 +1,259 @@
+import { Election, County, Result, Votes, Metrics } from "../../requested-data/requested-data-models";
```

Comment:
Not relevant to this PR but at some point we really should rename County to District. 🤷‍♂️